themes/src/main/resources-community/theme/base/admin/messages/admin-messages_de.properties

```diff
@@ -0,0 +1,1513 @@
+#client-scopes.evaluate.scopes.assigned=Selected Optional Client Scopes
+#client-scopes.evaluate.scopes.assigned.tooltip=Selected Optional Client Scopes, which will be used when issuing access token for this client. You can see above what value of OAuth Scope Parameter need to be used when you want to have these optional client scopes applied when the initial OpenID Connect Authentication request will be sent from your client adapter
+#client-scopes.evaluate.scopes.effective=Effective Client Scopes
+#client-scopes.evaluate.scopes.effective.tooltip=Contains all default client scopes and selected optional scopes. All protocol mappers and role scope mappings of all those client scopes will be used when generating access token issued for your client
+#client-scopes.evaluate.user.tooltip=Optionally select user, for whom the example access token will be generated. If you don't select any user, then example access token won't be generated during evaluation
+#send-evaluation-request=Evaluate
+#send-evaluation-request.tooltip=Click this to see all protocol mappers and role scope mappings, which will be used when issuing access token for this client. It will also optionally generate example access token in case that some user was selected
+
+#evaluated-protocol-mappers=Effective Protocol Mappers
+#evaluated-protocol-mappers.tooltip=Allow you to see all effective protocol mappers, which will be used when issuing token for this client. Contains also protocol mappers of selected optional client scopes. For each protocol mapper, you can see from which client scope it is inherited from
+#evaluated-roles=Effective Role Scope Mappings
+#evaluated-roles.tooltip=Allow you to see all effective roles scope mappings, which will be used when issuing token for this client. Contains also role scope mappings of selected optional client scopes
+#parent-client-scope=Parent Client Scope
+#client-scopes.evaluate.not-granted-roles=Not Granted Roles
+#client-scopes.evaluate.not-granted-roles.tooltip=Client doesn't have scope mappings for these roles. Those roles won't be in the access token issued to this client even if authenticated user is member of them
+#client-scopes.evaluate.granted-realm-effective-roles=Granted Effective Realm Roles
+#client-scopes.evaluate.granted-realm-effective-roles.tooltip=Client has scope mappings for these roles. Those roles will be in the access token issued to this client if authenticated user is member of them
+#client-scopes.evaluate.granted-client-effective-roles=Granted Effective Client Roles
+#generated-access-token=Generated Access Token
+#generated-access-token.tooltip=See the example token, which will be generated and sent to the client when selected user is authenticated. You can see claims and roles, which the token will contain based on the effective protocol mappers and role scope mappings and also based on the claims/roles assigned to user himself
+
+manage=Verwalten
+authentication=Authentifikation
```

Comment:
Is this really "Authentifikation" or maybe more "Authentifizierung"? The two words do not mean the same, see https://de.wikipedia.org/wiki/Authentifizierung .